client_linux_e2e_test.go

```diff
@@ -0,0 +1,57 @@
+//go:build integration
+
+// Tests in this file differ from those in the other files, as these
+// will actually connect over netlink and configure IPVS in the kernel, then
+// check the results against the "ipvsadm" tool.
+//
+// This may break your system's network configuration.
+//
+// You have been warned.
+
+package ipvs
```

Comment:
For integration style tests I would do `ipvs_test` so you're forced to use the exported APIs.